src/test/Makefile

```diff
@@ -748,7 +767,7 @@ $(OBJECT_DIR)/$1/%.c.o: $(TARGET_DIR)/$(TARGET)/%.c
 endif
 
 $(OBJECT_DIR)/$1/$(basename $1).o: $(TEST_DIR)/$(basename $1).cc
-	@echo "compiling $$<" "$(STDOUT)"
+	@echo "compiling $$<" "$(STDOUT)"s
```

Comment:
Is this a typo ?

Reply:
yes, editor mishap saving the file.  fixed.